src/main/java/com/hobbycollection/HobbyCollectionRESTController.java

```diff
@@ -8,6 +8,11 @@
 @RestController
 public class HobbyCollectionRESTController {
 
+    /**
+     *
+     * @param id
+     * @return URL
+     */
     @GetMapping("/api/Collection/fetchByid")
     public Collection collectionFetchById(@RequestParam(value="id") int id){
         return new Collection("Test", "http://something","Test Description",1 );
```

Comment:
Added JavaDoc here for collectionFetchByID/

Minor change, was something I saw when I was going through the code. Need to add a description of what the method does.

Citation: [https://www.tutorialspoint.com/java/java_documentation.htm](https://www.tutorialspoint.com/java/java_documentation.htm)

Reply:
This JavaDoc isn't adding much value.  If you add the JavaDoc, add the description to it as well.